src/AppBundle/Resources/views/Admin/Index/editUser.html.twig

```diff
@@ -19,8 +19,7 @@
 
     {% set numberOfClients = user.clients | length %}
     {% set firstClient = numberOfClients > 0 ? (user.clients | first) : null %}
-    {% set reports = firstClient ? firstClient.reports : [] %}
-    {% set reportsCount = reports | length %}
+    {% set reportsCount = firstClient ? firstClient.totalReportCount : 0 %}
```

Comment:
never liked the firstClient being used either. But that can be looked at another time